plugin/untested/v1/gogetter/go.mod

```diff
@@ -34,7 +34,3 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.13.7 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
-
```

Comment:
This file needs to be added to `go.work` as well

Reply:
Sorry, I add.